.github/workflows/process-load-test-results-template.yml

```diff
@@ -58,9 +58,21 @@ jobs:
           done
         env:
           GITHUB_TOKEN: ${{ secrets.ballerina_bot_token }}
+      - name: Check the existence of load-test-results branch
+        if: ${{ steps.processResults.outputs.hasPassed }}
```

Comment:
Can there be any cases where the `load-test-results` branch is created even if the `hasPassed` is set to false? (If the previous step fails inbetween those 2 steps it can happen IMO). So is that an option to clear the existing branch regardless of the output from the previous step?

Reply:
In the previous step we create a branch locally but we push the branch in the next step. We only push the changes when the load tests are passing, otherwise we will not push. So I have added this branch removal with the same check, so that it will be removed only if we want to push the same branch in the next step

Reply:
got it. makes sense